### PR TITLE
Student home page is created

### LINKED ISSUE #269: Student home page
Create a new page under `/home/student` for the student. The page should fetch the events for the user and display them.  If there are no events, it should look like this: 
image

If there are events, they should be in the list like this:
image

Add the link `Find events` that links to `/events` at the bottom of the content in _both_ pages. This is missing in Figma, but just add it as a simple link.

### PR DESCRIPTION
**Pull request by AnilMula**
# Description

The PR is to create a student home page and list all the events for the user.
NOTE: only link is from endpoints and the remaining week, datetime and mentor name are hardcoded

Fixes #269 

# How to test?

Tested in local machine by running the project and giving the URL in the browser http://localhost:3000/home/student

here is the test screenshot and gif

1.when there are no events 
Untitled

2. when there are events
chrome-capture (2)

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/App.js [hunk 1]
```diff
@@ -53,6 +54,9 @@ function App() {
           <RegistrationSuccessPage />
         </Route>
 
+        <Route exact path="/home/student">
```
Comment (tpetrina) [inline src/client/App.js:57]: Should be `AuthenticatedRoute`.

File: src/client/containers/StudentHome/StudentHome.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import Footer from '../../components/footer/footer';
+
+function StudentHome() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [events, setEvents] = useState([]);
+  const cardTitle = 'No events to show here';
+  const cardText =
+    'There are no events to show here maybe  you did not book any event yet. Please go to events page and book an event';
+  useEffect(() => {
+    axios.get('/api/events').then((response) => {
```
Comment (tpetrina) [inline src/client/containers/StudentHome/StudentHome.js:22]: This is the wrong URL, find out the correct URL by asking colleagues or going through closed issues. This should be a list of events that the user has applied for.

Comment (tpetrina) [inline src/client/containers/StudentHome/StudentHome.js:22]: Also, don't use `axios`, use fetch from the #255 .

File: src/client/containers/StudentHome/StudentHome.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                  link={event.venue}
+                >
+                  Register Yourself
+                </CardWithEventsOrStudyGroups>
+              )}
+            </div>
+          ))}
+        </div>
+      )}
+
```
Comment (tpetrina) [inline src/client/containers/StudentHome/StudentHome.js:77]: You forgot that link from the issue.

File: src/client/containers/StudentHome/StudentHome.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+              {event.organizationId === 1 && (
+                <CardWithEventsOrStudyGroups
+                  path={HYFLogo}
+                  title="HYF - Week 46"
+                  datetime="Wednesday | 11 Nov 20|10:00 CEST"
+                  mentor="Benjamin"
+                  link={event.venue}
+                >
+                  Register Yourself
+                </CardWithEventsOrStudyGroups>
```
Comment (tpetrina) [inline src/client/containers/StudentHome/StudentHome.js:49]: I'll let this one slide, but surely the data here should come from the API. Not all mentors are named Benjamin and not all events are on Tuseday last year :)

**Review by tpetrina — CHANGES_REQUESTED** [review]
Great start, needs a bit more work.

File: src/client/containers/StudentHome/StudentHome.js (hunk 2 above)
Comment (AnilMula) [reply src/client/containers/StudentHome/StudentHome.js:22]: yes true,  this is the URL 
​/eventsUsers​/{userID}​/{role}.  can you give me an idea to get username. if i get user name i can get userId and role. i am a bit confused in this authentication part

File: src/server/api/controllers/events.controller.js [hunk 5]
```diff
@@ -19,7 +19,15 @@ const createEvent = async (body) => {
 const getEvents = async () => {
   try {
     return await knex('events')
-      .select('events.id', 'events.event_type', 'event_date', 'venue')
+      .select(
+        'events.id',
+        'events.event_type',
+        'event_date',
+        'venue',
+        'max_participants',
```
Comment (tpetrina) [inline src/server/api/controllers/events.controller.js:27]:
```suggestion
        'maxParticipants',
```

**Pull request merged**